Passwords/muslimMatch.txt

```diff
@@ -95069,4 +95068,4 @@ zzzzzzz
 zzzzzzzz
 zzzzzzzzzz
 zzzzzzzzzzzz
-~~@@~@
\ No newline at end of file
+~~@@~@
```

Comment:
wait what's the diff here

Reply:
line ending, likely

Reply:
oh so it's probably unintentional.

i'm gonna request a change.

Reply:
It's a vim bug, I guess

Reply:
@vitalkanev Not a bug, but a feature. You can disable it by `:set nofixendofline`.